modules/virt-troubleshooting-vm-import.adoc

```diff
@@ -33,6 +35,7 @@ You can check the VM Import Controller Pod log for errors.
 
 . Obtain the VM Import Controller Pod name by running the following command:
 +
+[source,terminal]
 ----
 $ oc get pods -n <project> | grep import <1>
```

Comment:
Same as above -- the `vm-import-controller....` is example output.

Reply:
Done